example/nomad_acl_test.tf

```diff
@@ -1,3 +1,8 @@
+provider "vault" {
+  address = "http://127.0.0.1:8200"
+  token = "master"
+}
+
```

Comment:
WDYT to rename this file to `providers.tf`

Reply:
I can do that, but then I think we should move the data-obj to main.tf. Or what do you think?

Reply:
If you mean `vault_generic_secret`([link](https://github.com/fredrikhgrelland/terraform-nomad-presto/blob/sync/example/nomad_acl_test.tf#L6-L10)), not necessary. I think `vault_generic_secret` locates in the context of communicating terraform with vault to be able to fetch newly generated token for nomad.

Reply:
Okey, then i'll just rename `nomad_acl_test.tf` :point_right: `providers.tf` for now.